### PR TITLE
Import Utility for Evaluations

### PR DESCRIPTION
**Pull request by bbengfort**
### Scope of changes

Updates the models with fields recommended by @pdeziel and creates an import utility to load the data.

### Type of change

- [ ] bug fix
- [x] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria

Can we start uploading data into the app after this PR is merged and the app is deployed?

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ] I have updated the dependencies list
- [ ] I have added new test fixtures as needed to support added tests
- [ ] I have added or updated the documentation
- [x] Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] To the best of my ability, I believe that this PR represents a good solution to the specified problem and that it should be merged into the main code base.
- [ ] These fields are sufficient for us to start uploading model data today.

### CODE REVIEW

File: parley/models.py [hunk 1]
```diff
@@ -78,12 +78,30 @@ class LLM(BaseModel):
...
+        null=True, default=None, blank=True,
+        help_text="The standardized generation config of the model",
+    )
+
+    quantization_info = models.JSONField(
+        null=True, default=None, blank=True,
+        help_text="Information about the quantization of the model, if any",
+    )
+
+    tokenizer_info = models.JSONField(
```
Comment (pdeziel) [inline parley/models.py]:
```suggestion
    tokenizer_config = models.JSONField(
```

File: parley/models.py [hunk 2]
```diff
@@ -78,12 +78,30 @@ class LLM(BaseModel):
     )
 
     description = models.TextField(
-        null=True,
-        blank=True,
-        default=None,
+        null=True, blank=True, default=None,
         help_text="Any notes or other descriptive information about the model or training process",
     )
```
Comment (pdeziel) [inline parley/models.py:84]: I think we should have a model config as well so we know how to instantiate the model. We may also need an `is_adapter_model` boolean to tell if the model is a base model or a LoRA.

File: parley/models.py (hunk 1 above)
Comment (pdeziel) [reply parley/models.py]: Just to make this consistent with HuggingFace terminology.

**Review by pdeziel — COMMENTED** [review]
Thanks, I added a few more suggestions to make the LLM definition more complete. So is the remaining work to add a JSON utility to upload the models, prompts, and responses in a combined JSON file?

**Comment by bbengfort** [comment]
> Thanks, I added a few more suggestions to make the LLM definition more complete. So is the remaining work to add a JSON utility to upload the models, prompts, and responses in a combined JSON file?

Yes, I'm working on that uploader right now.

**Comment by bbengfort** [comment]
@pdeziel I added those fields to the models and also added validation fields for the prompt and responses.

**Pull request merged**